### PR TITLE
Expose lookupParam helper function.

### PR DESCRIPTION
**Pull request by tmcgilchrist**
400 or 500 for this? 

@charleso @reiddraper 
https://github.com/helium/airship/pull/63

### CODE REVIEW

File: src/Airship/Internal/Helpers.hs [hunk 1]
```diff
@@ -124,3 +124,13 @@ getQuip = do
 
 traceHeader :: [Text] -> ByteString
 traceHeader = encodeUtf8 . intercalate ","
+
+-- | Lookup routing parameter and return 400 Bad Request if not found.
+-- Not finding the paramter usually means the route doesn't match what
+-- the resource is expecting.
+lookupParam :: Monad m => Text -> Webmachine m Text
+lookupParam p = lookupParam' p >>= maybe (halt HTTP.status400) pure
```
Comment (charleso) [inline src/Airship/Internal/Helpers.hs]: As mentioned on the commit, I actually think this should be `500`

**Comment by charleso** [comment]
:100: from me obviously :)

**Comment by reiddraper** [comment]
Agree, should be 500 here.

**Comment by tmcgilchrist** [comment]
Changed to 500 and updated the docs accordingly. 
Will merge if it's all Green.

**Pull request merged**